lang/python/sqlparse/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sqlparse
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+PKG_LICENSE:=BSD
+PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
+
+PKG_SOURCE:=sqlparse-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/s/sqlparse
+PKG_HASH:=7c3dca29c022744e95b547e867cee89f4fce4373f3549ccd8797d8eb52cdb873
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/sqlparse-$(BUILD_VARIANT)-$(PKG_VERSION)
+PKG_UNPACK=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/sqlparse
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Non-validating SQL parser module.
+  URL:=https://github.com/andialbrecht/sqlparse
+  DEPENDS:=python3-base 
+  VARIANT:=python3
+endef
+
+define Package/sqlparse/description
+  A non-validating SQL parser module. It provides support for parsing, splitting and formatting SQL statements.
+endef
+
+$(eval $(call Py3Package,sqlparse))
+$(eval $(call BuildPackage,sqlparse))
```

Comment:
not sure here;
maybe let's rename this to `python3-sqlparse` or at least `python-sqlparse`

Reply:
You are right, `python3-sqlparse` is better, I did change that.

All proposed packages are only for Python 3as the python/README.md says: 
> From 31 May 2019 onward, absolutely no new Python 2 packages will be accepted

Then, I would wait for the update of `python-django` without introducing a new package. (There would be also a fallback possibility as `seafile` is using the Python 2 version of django: We could use the 2.2.5 version of django only for `python3-django`. But, this is maybe too confusing. At least this confusion could not be longer than until "The Snap" off the Python 2 packages.)

And you are right, too, that the `django2restframework` will then be obsolete. I would remove it then.